pytest_operator/plugin.py

```diff
@@ -0,0 +1,108 @@
+import os
+import pytest
+import asyncio
+from pathlib import Path
+from juju.model import Model as JujuModel
+from aioify import aioify
+from subprocess import run
+
+
+def pytest_addoption(parser):
+
+    parser.addoption("--controller", action="store", help="Juju controller to use")
+
+    parser.addoption("--model", action="store", help="Juju model to use")
+
+    parser.addoption(
+        "--cloud", action="store", help="Juju cloud to use"
+    )
+
+
+class ToolsObj:
+    """Utility class for accessing juju related tools"""
+
+    def __init__(self, request):
+        self._request = request
+
+    async def _load(self):
```

Comment:
You're indicating this is 'private' but you don't call it from __init__. If this is required to use the class, it should either be called from init or made non-private to indicate it's expected to be called.